index.js

```diff
@@ -7,6 +7,14 @@ const { flow } = require('./lib/configs/flow')
 const { recommended } = require('./lib/configs/recommended')
 
 module.exports = {
+    prettier: {
+        arrowParens: 'avoid',
+        trailingComma: 'all',
+        tabWidth: 4,
+        semi: false,
+        singleQuote: true,
+        printWidth: 100,
```

Comment:
printWidth is `120` in `insights` and `datatap` - this should definitely be the default! Do we really have `100` in presense?

```suggestion
        printWidth: 120,
```